test/eq.test.js

```diff
@@ -36,10 +36,7 @@ const require = createRequire(
             });
 
             it("should return boolean", function () {
-                const returnValue = true,
-                returnValue2 = false;
-                assert.isBoolean(eq(returnValue), "correct answer")
-                assert.isBoolean(eq(returnValue2),"wrong answer")
+                assert.isBoolean(eq());    
```

Comment:
You should give the function call some parameters (for example, eq("abba", "abba"))